submarine-server/server-submitter/submitter-yarn/pom.xml

```diff
@@ -27,7 +27,6 @@
   <artifactId>submarine-submitter-yarn</artifactId>
   <version>0.6.0-SNAPSHOT</version>
```

Comment:
In addition, The community has decide to support k8s more time.
so there is not need speed time on yarn in the feature.